### DISCUSSION TITLE
B站的倒卖狗又在卖了！

### DISCUSSION THREAD

**Discussion by BlankTMing** [pjy612/SteamManifestCache#107] [discussion]
不过这次一看就是用的heyong5454的仓库，以后还是吧key文件加密一下

**Comment by BlankTMing** [comment]
有两个倒卖狗在卖，沧海颐粟，11steam，长夜远行

**Reply by pjy612** [reply]
是啊 =。= 
11steam 好像是 别的一套类似 tools 的东西。
沧海颐粟 和 长夜远行 一个人。

反正我只求别拿咱的卖就行，很坏心情... 2333

**Reply by pjy612** [reply]
heyong 的库 好像更新频率降低了

**Reply by pjy612** [reply]
现在 沧海 听反馈 应该演示的是 ikun 的 OneKey

**Reply by pjy612** [reply]
纠正 11steam 也是 套的 tools.

**Comment by BlankTMing** [comment]
> 现在 沧海 听反馈 应该演示的是 ikun 的 OneKey

看到了，这人是一点都不会自己写的全是套壳

**Comment by ZEERDEER** [comment]
用AI都能写出来工具，盗卖的真是杂种

**Comment by mengqian-666** [comment]
的确，坑了很多人的马内（至少1000吧）
